### PR TITLE
BookStore: Milestone 5

### PR DESCRIPTION
**Pull request by jebitok-dev**
In this milestone I implemented the Bookstore CMS design by:-
- adding the Navbar component 
- modifying the book component to correspond with design 
- adding styles

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -24,4 +24,4 @@ code {
     Consolas,
     'Courier New',
     monospace;
-}
+} */
```
Comment (mimipeshy) [inline src/index.css:27]: - [ ] Remove commented out code

**Comment by mimipeshy** [comment]
## STATUS : CHANGES REQUESTED 🔴 ♻️ 
Hello, @jebitok-dev 👋🏾 👋🏾 congratulations on making it this far 😄 . 

#### Highlights

- You have worked so hard on the UI it looks great ✔️ 

#### To improve
Let's improve on the following sections⬇️ 
- [x] Index.css has commented out code, kindly remove it. You can opt to use `git history` and you can trace the commit you want this way, you keep your code clean
- [ ] There is a small bug in your app, I am unable to add a new book in the form provided. Instead, I get the following error in the console ⬇️ 
image

- [ ] Let's clean up our CSS a little bit more and keep it DRY 😄 Check out my inline comments below ⬇️ 

That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

File: src/styles/App.css [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+  line-height: normal;
+}
+
+.App {
+  width: 90%;
+}
+
+.navbar {
+  width: 100%;
+  display: flex;
```
Comment (mimipeshy) [inline src/styles/App.css]: - [x] Display `flex` has been used 6 times. You can create a CSS class for this and just refer to the class whenever you need the properties
eg
`
d-flex{
display:flex
}
So you just need to call display: d-flex as many times as you want

File: src/styles/App.css [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+  justify-content: space-around;
+  align-items: center;
```
Comment (mimipeshy) [inline src/styles/App.css]: - [x] Align-items :center has also been repeated. Let's keep it DRY by applying the concepts of display:flex i demonstrated above

File: src/styles/App.css [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+  letter-spacing: -0.2px;
+  color: var(--black);
+}
+
+.book-item .author {
+  width: 107px;
+  height: 19px;
+  margin: 4px 0 0;
+  font-size: 14px;
+  font-weight: 300;
```
Comment (mimipeshy) [inline src/styles/App.css]: - [x] Font-weight:300 has been repeated a lot too. Lets keep this DRY as well 😄

File: src/styles/App.css [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+  align-items: center;
+  margin: auto;
+}
+
+.book-title {
+  width: 50%;
+  margin-left: 100px;
+  height: 45px;
+  padding: 13px 571px 13px 17px;
+  border-radius: 4px;
```
Comment (mimipeshy) [inline src/styles/App.css]: - [ ] Elements that have the same property of border-radius can be defined under one class eg
``
book-title,
book-category{
border-radius :4px
}
``

Comment (jebitok-dev) [reply src/styles/App.css]: Thank you for highlighting those changes to be made

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED👍️👍️👍️👍️👍️👍️**

Great work making the required changes👏️👏️👏️

You can merge👍️👍️

Happy coding💻️💻️💻️💻️

**Pull request merged**